src/js/core/directives/ui-grid-header-cell.js

```diff
@@ -62,6 +63,8 @@
 
             $scope.grid = uiGridCtrl.grid;
 
+            $scope.showColMenuButton = $scope.grid.options.enableColumnMenus && !$scope.col.isRowHeader  && $scope.col.colDef.enableColumnMenu !== false;
```

Comment:
It is redundant to check if $scope.grid.options.enableColumnMenus is truthy and then immediately check if it is different than false. I suggesting doing just the false check

Reply:
$scope.grid.options.enableColumnMenus and $scope.col.colDef.enableColumnMenu are not the same. The first is general and the second is column specific.

Reply:
whoops, I didn't see the s. good point

Reply:
Would it be possible to add some unit tests for this line?